rucio-server/values.yaml

```diff
@@ -113,6 +113,17 @@ metricsExporterResources:
    cpu: 20m
    memory: 20Mi
 
+ftsRenewal:
+  enabled: 0
+  schedule: "12 */6 * * *"
```

Comment:
Please change the default schedule based on the comment above. Those taking the defaults for both will experience transient problems.